deploy/charts/harvester-crd/README.md

```diff
@@ -0,0 +1,58 @@
+# Harvester CRD Helm Chart
+
+## Chart Details
+
+This chart includes following CRDs:
+
+- Harvester (Automatically generated by [this script](https://github.com/harvester/harvester/blob/master/scripts/generate-manifest))
+- KubeVirt
+- csi-snapshotter
+- whereabouts
+
+## Introduction
+
+Harvester depends on several components that provide the CRDs it needs. However, most of these CRDs must be created manually. To simplify management, Harvester uses an [independent folder to manage all CRDs](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/#method-2-separate-charts).
+
+That said, not all CRDs can be managed by Helm. For example, the `cdi` CRD is managed by KubeVirt, but KubeVirt does not create the CRD automatically. Therefore, Harvester must create it manually. These CRDs are applied in the [`upgrade_manifests.sh`](https://github.com/harvester/harvester/blob/50da36ac3b751c1a1dbfc8d25e5499a4c6216450/package/upgrade/upgrade_manifests.sh#L1358) script.
```

Comment:
IIRC, the CDI CRD is not managed by KubeVirt (though it's under the `cdi.kubevirt.io` group). It would be better to clarify this.

Reply:
I think my words might confuse. I mean the cdi-operator will delete the older version of cdi CRD, but they don't create CRD at the beginning. I'll revise this part and make it more clear, thanks!